polkadot/runtime/common/src/claims.rs

```diff
@@ -620,7 +618,7 @@ where
 	}
 }
 
-impl<T: Config + Send + Sync> SignedExtension for PrevalidateAttests<T>
+impl<T: Config> SignedExtension for PrevalidateAttests<T>
```

Comment:
If you would just kept the bounds here, it would have been enough. 

But we can go with the phantomdata trick.